tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -568,6 +570,12 @@ def test_all(testerchain, token, escrow, policy_manager, adjudicator, user_escro
     tx = user_escrow_proxy_1.functions.confirmActivity().transact({'from': ursula3})
     testerchain.wait_for_receipt(tx)
 
+    # Turn off re-stake for Ursula1
```

Comment:
@szotov -  So, Restaking can be enabled and disables only where there are *no active stakes*, is that correct?

Reply:
No, at any time if not locked. Example at line 931

Reply:
:+1: